libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -106,3 +106,43 @@ export const NoDismiss: Story = {
     template: `<kirby-x-image-banner ${argsToTemplate(args)}></kirby-x-image-banner>`,
   }),
 };
+
+export const containerQueries: Story = {
```

Comment:
Please add jsdoc similar to other stories, explaining what the user sees in the example. We use this for documentation in storybook.

Reply:
Something about how the component adapts to the containers width, and thus should be plug and play with the kirby css grid utilities.